android/src/com/frostwire/android/gui/adapters/menu/SeedAction.java

```diff
@@ -122,6 +122,7 @@ protected void onClick(Context context) {
         // in case user seeds only on wifi and there's no wifi, we let them know what will occur.
         if (seedingOnlyOnWifiButNoWifi()) {
             showNoWifiInformationDialog();
+            return;
```

Comment:
this fixes a bug on which upon denying the user the seeding action, the lack of this `return` call would still create the torrent and add it in `Finished` state, and most likely take the user away from the My Files screen and into Transfers.